casepro/contacts/models.py

```diff
@@ -190,7 +190,7 @@ def get_or_create_from_urn(cls, org, urn, name=None):
         """
         contact = cls.objects.filter(urns__contains=[urn]).first()
         if not contact:
-            contact = cls.objects.create(org=org, name=name, urns=[urn], is_stub=True)
+            contact = cls.objects.create(org=org, name=name, urns=[urn], is_stub=False)
```

Comment:
We cannot create a new case with a stub contact: https://github.com/praekelt/casepro/blob/069088371eee6cd816ab1fe79d566b01f4cc6446/casepro/contacts/models.py#L230-L231